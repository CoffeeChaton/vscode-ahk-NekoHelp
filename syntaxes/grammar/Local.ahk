;@ahk-neko-format-inline-spacing-ignore-start
Local ;comment 
static ; comment
global ;comment

Local  ;comment 
static  ; comment
global  ;comment
;@ahk-neko-format-inline-spacing-ignore-end
