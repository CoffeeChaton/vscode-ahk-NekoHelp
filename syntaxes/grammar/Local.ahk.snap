>;@ahk-neko-format-inline-spacing-ignore-start
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>Local ;comment 
#^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#     ^ source.ahk meta.comment.line.semicolon.ahk
#      ^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>static ; comment
#^^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#      ^ source.ahk meta.comment.line.semicolon.ahk
#       ^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>global ;comment
#^^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#      ^ source.ahk meta.comment.line.semicolon.ahk
#       ^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>Local  ;comment 
#^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#     ^ source.ahk
#      ^ source.ahk meta.comment.line.semicolon.ahk
#       ^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>static  ; comment
#^^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#      ^ source.ahk
#       ^ source.ahk meta.comment.line.semicolon.ahk
#        ^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>global  ;comment
#^^^^^^ source.ahk keyword.other.operator.hasdoc.ahk
#      ^ source.ahk
#       ^ source.ahk meta.comment.line.semicolon.ahk
#        ^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>;@ahk-neko-format-inline-spacing-ignore-end
#^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#          ^ source.ahk comment.line.semicolon.ahk
#           ^^^^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
#                 ^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
#                                        ^^^ source.ahk comment.line.semicolon.ahk keyword.other.ahk
>